apps/token/src/i18n/translations/dev.json

```diff
@@ -512,10 +512,11 @@
   "unsupportedChainIdError": "You're connected to an unsupported network",
   "userRejectionError": "Please authorise this website to access your Ethereum account",
   "unknownEthereumConnectionError": "An unknown error occurred. Check the console in your browser's web developer tools for more details",
-  "stakingDescription1": "Associate VEGA tokens from your Ethereum wallet to a Vega wallet/key to nominate a validator",
-  "stakingDescription2": "Validators and their nominators are rewared every 24 hours for securing the network with a cut of trading fees (and rewards from the Vega treasury). Staking rewards are paid into your Vega wallet after an epoch ends.",
-  "stakingDescription3": "The amount that a validator, and its nominators receives is proportionate to the amount staked on it, however a validator can be penalised during an epoch for a number of reasons. In this case the validator and its nominator will forfeit some or all of its reward from that epoch",
-  "stakingDescription4": "A validator will receive a higher share of the pot than its nominators. This share is the same for all validators and is set by a network parameter.",
+  "stakingDescriptionTitle": "How does staking on Vega work?",
+  "stakingDescription1": "1. VEGA is an ERC20 token. Associate it with a Vega wallet using a staking bridge",
+  "stakingDescription2": "2. Use your Vega wallet to nominate a validator",
```

Comment:
Comment (semi-non-blocking?): Really you're using the token dApp to nominate a validator as you can't do it from within the wallet. But am I missing the gist of what this is trying to convey?

Reply:
That’s clearer! @campbellssource @sam-keen